src/Gateway/MolliePaymentGateway.php

```diff
@@ -453,23 +453,23 @@ public function is_available(): bool
                 $currency = get_woocommerce_currency();
             }
 
-            $billing_country = WC()->customer ? WC()->customer->get_billing_country() : wc_get_base_location()['country'];
+            $billingCountry = WC()->customer && !empty(WC()->customer->get_billing_country())? WC()->customer->get_billing_country() : wc_get_base_location()['country'];
```

Comment:
I think it would make sense to put this in a separate method. There is quite some "complexity" here with the `empty` + ternary operation. This can be written in a more verbose and understandable fashion if there's a dedicated method. Also you can document the intention in a docblock